File: Easy/06.LetterCapitalize/solve.rb
def LetterCapitalize(str)
    
  # code goes here
  	result = str.split().map { |word| word = word[0].upcase + word[1..word.length-1] }
	result.join(" ")
         
end
   
# keep this function call here    
puts LetterCapitalize(STDIN.gets)

File: Medium/09.BinaryConvertor/solve.py
def BinaryConverter(s):
	result = [int(x) for x in s[::-1]]
	base = 2
	r = 0
	
	for i in range(0, len(result)):
		if result[i] == 1:
			r += base ** i
	return r
    
# keep this function call here  
print BinaryConverter(raw_input())